docs/components/receive.md

```diff
@@ -30,6 +30,15 @@ thanos receive \
     --objstore.config-file "bucket.yml"
 ```
 
+The example of remote_write Prometheus configuration:
```

Comment:
```suggestion
The example of `remote_write` Prometheus configuration:
```

Reply:
Can we please also add a link to the Prometheus documentation for further details?

Reply:
The link is on top of this page.